stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/Message.java

```diff
@@ -381,7 +381,7 @@ private void updateBMapValue(BType bType, MapValue<String, Object> bMapValue,
         Descriptors.OneofDescriptor oneofDescriptor = fieldDescriptor.getContainingOneof();
         String msgType = oneofDescriptor.getContainingType().getName() + "_" + toCamelCase
                 (fieldDescriptor.getName());
-        MapValue<String, Object> bMsg = new MapValueImpl<>(new BRecordType(msgType, bType.getPackage(), 0, true));
+        MapValue<String, Object> bMsg = new MapValueImpl<>(new BRecordType(msgType, bType.getPackage(), 0, true, 0));
```

Comment:
Do we know what this is used for? Wondering if we should add TypeFlags.ANYDATA, TypeFlags.PURETYPE?